packages/lab/src/stepped-tracker/Step.tsx

```diff
@@ -0,0 +1,178 @@
+import {
+  type ButtonProps,
+  makePrefixer,
+  useControlled,
+  useId,
+} from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import clsx from "clsx";
+import {
+  type CSSProperties,
+  type ComponentProps,
+  type ReactNode,
+  useContext,
+  useEffect,
+} from "react";
+
+import { StepConnector } from "./Step.Connector";
+import { StepDescription } from "./Step.Description";
+import { StepExpandTrigger } from "./Step.ExpandTrigger";
+import { StepIcon } from "./Step.Icon";
+import { StepLabel } from "./Step.Label";
+import stepCSS from "./Step.css";
+import { SteppedTracker } from "./SteppedTracker";
+import { DepthContext } from "./SteppedTracker.Provider";
+
+export namespace Step {
+  export interface Props extends Omit<ComponentProps<"li">, "onToggle"> {
+    id?: string;
+    label?: ReactNode;
+    description?: ReactNode;
+    status?: Step.Status;
+    stage?: Step.Stage;
+    expanded?: boolean;
+    defaultExpanded?: boolean;
+    onToggle?: ButtonProps["onClick"];
+    substeps?: Step.Props[];
+    children?: ReactNode;
+  }
+
+  export type Status = "warning" | "error";
+
+  export type Stage =
+    | "pending"
+    | "locked"
+    | "completed"
+    | "inprogress"
+    | "active";
+
+  /**
+   * noun: elucidation; pl. noun: elucidations
+   * explanation that makes something clear; 🤔
+   */
+  export type Elucidation = Status | Stage;
```

Comment:
New language needs alignment

Reply:
This is internal use only, as the type suggest it is a union type of status and stage. Since only one can be applied at a time, I require some type to have strongly typed variable and comply with a11y purposes. It is a very uncommonly used word, doubt that it is going to pop up in the future.

Reply:
its no longer there, i've removed it